### PR TITLE
[IMP] crm: utilize activities for sales teams 17

### PR DESCRIPTION
**Pull request by jero-odoo**
Updating the CRM utilize activities doc for 17.
Adding content for Activity plans feature introduced for V17

Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=3864366&access_token=d0e70da6-6922-449c-a028-b386f4b63721

### CODE REVIEW

**Comment by jero-odoo** [comment]
@StraubCreative this is ready for review. Thanks!

File: content/applications/sales/crm/optimize/utilize_activities.rst [hunk 1]
```diff
@@ -159,67 +161,92 @@ past due activities.
 The pipeline is most effective when it is kept up-to-date and accurate to the interactions it is
 tracking.
 
+.. _crm/activity-plans:
+
 Activity plans
---------------
+==============
+
+*Activity Plans* are preconfigured sequences of activities that can be triggered with one click,
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/utilize_activities.rst:169]: Capitalization, please pick one (Activity plans vs. Activity Plans)

File: content/applications/sales/crm/optimize/utilize_activities.rst [hunk 2]
```diff
@@ -159,67 +161,92 @@ past due activities.
...
+and replicated with little to no manual setup. Activity plans can be customized for various
+workflows, with multiple deadlines and responsible users assigned in a single plan.
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/utilize_activities.rst:171]: Consider condensing this first paragraph to necessary and specific information. Seems a bit wordy, borderline sales-y.

File: content/applications/sales/crm/optimize/utilize_activities.rst [hunk 3]
```diff
@@ -159,67 +161,92 @@ past due activities.
...
+Create` tab, click :guilabel:`Add a line`.
+
+First, select an :guilabel:`Activity Type` from the drop-down menu. Click :guilabel:`Search More` to
+see a complete list of available activity types, or to create a :ref:`new one
+<crm/create-new-activity-type>`.
+
+Next, in the :guilabel:`Summary` field, enter any details to describe the specifics of the step. The
+contents of this field are included with the scheduled activity, and can be edited later.
+
+In the :guilabel:`Assignment` field, select one of the following options:
```
Comment (StraubCreative) [inline content/applications/sales/crm/optimize/utilize_activities.rst:186]: The wording here comes across as a bit short. The instructions are clear but it's not clear why we're doing anything. Consider expanding a bit.

As well, we say "First select an **Activity Type**..." however this isn't really the first step in the instructions so that threw me off (we're saying "First do this" after 2-3 steps before it)

**Review by StraubCreative — APPROVED** [review]
Hi @jero-odoo,
Good section addition here. Approving with comments. 
Can you take a look, and merge when ready? Thanks!
..
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+
